### ISSUE TITLE
Controller Support Stopped Working

### ISSUE THREAD

**Issue by MarkCashion** [tetrio/issues#396]
### Describe the bug/issue
Controller Support is not working. Even though I already _(by default)_ use a controller only custom key-bind layout. I can't even change the key-bind settings to any other controller button in general.

### Reproducing
1. Plug-in any type of controller
2. Try to do anything

### Expected behavior
Buttons work as if any other key-bind setting was used

### Your setup
 - Windows 10
 - TETR.IO desktop app V4.0.1

### Additional context
I changed the port my controller was plugged into, then I opened up Tetris Effect to see if it was my controller that was acting up, but the game recognized my controller & inputs, Steam & Windows recognized that I plugged in a controller too

**Comment by o5k** [comment]
Try closing Steam, it likes taking controller inputs away?

**Comment by MarkCashion** [comment]
Everything that registers my controller _(besides Windows, obviously)_ is closed, & yet the game still doesn't pick up the inputs

**Comment by ddderder** [comment]
The issue seems to be intermittend: I got the issue with my controller 5 days ago, was 2 days on vacation and it was "fixed" afterwards but reappeared 2 days later. I deleted and re-added the controllers and sometimes they work until I close and reopen the desktop app. Tried with 2 different controllers, one of them being an original xbox one controller, the other one is compatible and can be connected via cable and bluetooth, none of it works. The game recognizes the controller being available (green message box in bottom right screen, sometimes appearing twice or even 3 times) and also recognizes a disconnection of the controller but does not allow any keybindings at all. same goes for the browser version

**Comment by krysppy** [comment]
Hey! Deactivating vJoy (or in my case, uninstalling vJoy) worked for me.

When I had vJoy, I couldn't use the controller, and when the controller registered, a message appeared on the bottom right about 3 times. When I uninstalled it, all the problems disappeared.

**Comment by SilentastheWinds** [comment]
Hi, I'm having this issue as well, just installed the game, don't have vJoy, and the game detects my controller, but its not working.

**Comment by trogfield** [comment]
same problem. controller detected in win 10, all buttons work in testing mode, detected in game, cant bind keys

**Comment by nekoellie** [comment]
got the same issue with my 8bitDo SN30 pro+ connected as xinput. when i press anything, the game says two controllers simultaneously connect, yet trying to map the controller does nothing

**Comment by AeonWilder** [comment]
Having the same issue, seemingly randomly/inconsistently. Windows 10 with Xbox One Elite controller, and Dualshock 4 controller both produce the issue.

**Comment by trogfield** [comment]
> same problem. controller detected in win 10, all buttons work in testing mode, detected in game, cant bind keys

I think this happened when i used the external windows brand keyboard on a surface pro 7 with a wired usb xbox one controller. both are detected as a controller. The external keyboard is detected as a controller. The xbox controller is detected as a controller. only one or the other works at a time. sometimes both do, but usually only one could be plugged in.

**Comment by Nachocss** [comment]
Having issues with Dualshock as well in Windows 10.
I have tried to bypass the game through Steam Controller Configuration interface by using GloSC but same result, game detects the controller but doesn't handle any input from it.

**Comment by Rocketsniper** [comment]
Having the same issue in windows 10 with a XBox One controlller - in the game controllers window I see that it detects inputs from my controller, and I used to be able to bind the controller inputs to controls, but now tetr.io is not registering the controller. I have never had vJoy installed, but do have steam installed. I did restart my computer between success and failure, but restarting again results in failure today.  Windows game client has same issue as web client, and neither seem to be taking the input at all.
